turbo/generators/config.ts

```diff
@@ -16,6 +17,12 @@ export default function generator(plop) {
         message:
           "Enter a space separated list of dependencies you would like to install",
       },
+      {
+        type: "input",
+        name: "devdeps",
```

Comment:
I feel like this `devdeps` prompt might end up confusing people. For example, what if someone assumed that the `deps` prompt was meant to include all deps, including devDeps? Or maybe someone doesn't know (or care) about the difference between deps and devDeps? 🤷🏼 

I would ditch it, personally.

However, if we do want two prompts for deps, I would include something explicit in the first prompt like: `Enter a space separated list of dependencies you would like to install (you'll enter development dependencies next)`

Reply:
Yea delete this prompt.

I had it initially but it's confusing since we already install some devDeps by default

Reply:
I see you point and removed change accordingly!